doc/libfuzzer.md

```diff
@@ -0,0 +1,61 @@
+# libfuzzer
+
+## setup 
+
+### Requirements
+libFuzzer needs LLVM sanitizer support, 
+so this only works on x86-64 Linux, 
+x86-64 macOS and Apple-Silicon (aarch64) macOS for now. 
+This also needs a nightly compiler since 
+it uses some unstable command-line flags. 
+You'll also need a C++ compiler with C++11 support.
+
+
+
+### Installing
+
+rust-version|libfuzzer|td-shim|successful
+:-:|:-:|:-:|:-
+nightly-2022-02-08|0.11.0|staging|no (td-shim fails to compile in this version)
+nightly-2021-08-20|0.10.2|staging|yes
+nightly-2021-08-20|0.11.0|staging|no (libfuzzer failed to start)
```

Comment:
I'd consider adding a link to the issue you raised as someone is bound to try this.

Reply:
I added the link.